### ISSUE TITLE
Move from Flask-Babelex to Flask-Babel

### ISSUE THREAD

**Issue by zzacharo** [inveniosoftware/invenio-i18n#88]
[Flask-Babelex](https://github.com/mrjoes/flask-babelex) is deprecated and integrated in the [Flask-Babel](https://github.com/python-babel/flask-babel) module.

**Comment by chriz-uniba** [comment]
in a v11 `Pipfile.lock` I still see `flask-babelex` as dependency.

What is the status on this one?
@utnapischtim I remember, that you was working on some babel cleaning some day?
Was/is this related?

What's the status on this issue?

**Comment by utnapischtim** [comment]
@chriz-uniba yes v11 has still flask-babelex as dependency. the changes only apply to the v12.* versions

**Comment by chriz-uniba** [comment]
Okay, thanks - then I recheck with v12 as soon as time allows.

**Comment by chriz-uniba** [comment]
so I checked:
no flask-babelex in the `Pipfile.lock` of invenio-rdm version
 - 12.0.0b2.dev20
 - 12.0.0b2.dev0

this looks fixed to me and therefore, I think that this issue can be closed.
